app/views/sitters/show.html.erb

```diff
@@ -1,13 +1,9 @@
 <h1><%= @sitter.first_name %> <%= @sitter.last_name %></h1>
 
 <p>Details: <%= @sitter.details %></p>
-<p>Location: <%= @sitter.local_area %></p>
-<p>Verified: <%= @sitter.verified %></p>
-<hr>
-<%= render "requests/form", sitter: @sitter, request: @request %>
-<hr>
-<%= link_to 'Back to Sitters', sitters_path %>
+<p>Location: <%= @sitter.address %></p>
 <br>
-<%= link_to 'Home', root_path %>
+<%= render "requests/form", sitter: @sitter, request: @request %>
```

Comment:
This form should technically render "form" (the `_form.html.erb` in views/sitters), not "requests/form". It used to be requests/form because I wanted to reuse the form but I was having issues with routing. Have you verified that updating the form still works after your changes?